src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -1069,15 +1071,16 @@ subroutine ocn_tend_freq_filtered_thickness(tendPool, statePool, meshPool, timeL
         do i = 1, nEdgesOnCell(iCell)
           iEdge = edgesOnCell(i, iCell)
 
+          ! Note: changing loop limit to minLevelEdgeBot introduces non-bit-for-bit changes
           do k = 1, maxLevelEdgeTop(iEdge)
             flux = layerThickEdge(k, iEdge) * normalVelocity(k, iEdge) * dvEdge(iEdge) * edgeSignOnCell(i, iCell) * invAreaCell
             div_hu(k) = div_hu(k) - flux
             div_hu_btr = div_hu_btr - flux
           end do
         end do
 
-        totalThickness = sum(layerThickness(1:maxLevelCell(iCell),iCell))
-        do k = 1, maxLevelCell(iCell)
+        totalThickness = sum(layerThickness(minLevelCell(iCell):maxLevelCell(iCell),iCell))
+        do k = minLevelCell(iCell), maxLevelCell(iCell)
```

Comment:
not necessarily related to this place but what is the plan for cases when there are fewer than three active cells (minLevelCell within 3 of maxLevelCell). Will this be allowed?  If so, will the calls to ocn_tendency be skipped?  Something else?

Reply:
@cbegeman can respond but my feeling is that it's up to the code that actually computes the vertical coordinate (either at init or as it evolves in time) to enforce the 3-cell minimum.  That would be beyond the scope of this PR, of course.

Reply:
It's a great question, and one we haven't settled. We should continue the conversation [here](https://github.com/MPAS-Dev/MPAS-Model/pull/811/files#r574745570) and flesh out the design doc further on this point. It'll have to be more nuanced than skipping all calls to ocn_tendency.

Reply:
I totally agree this is way beyond this PR, so no need to address farther in this space.